Gemfile

```diff
@@ -1,4 +1,9 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in vagrant-ovirt.gemspec
-gemspec
+group :development do
+  gem 'vagrant', :git => 'git://github.com/mitchellh/vagrant.git'
```

Comment:
vagrant is no longer supported as a gem, and their latest gem is 1.5.0 as compared to the 1.7.2 release